server/views/components/instagram-thumbnail/index.njk

```diff
@@ -0,0 +1,31 @@
+<a href="{{ model.url }}" class="instagram-thumbnail"
+  data-track-click='{"screenName": "{{ model.screenName }}", "createdAt": "{{ model.createdAt }}", "postId": {{ model.id }}}'>
```

Comment:
> Can't harm
Adding things for people to have to filter through, irrespective of how simple it is, seems unnecessary?

But if we can see merit, that's alright, but could we perhaps hypothesise why this data might be useful?

Isn't the `screenName` always going to be ours, and `createdAt` just be the latest posts, making it useless to aggregate or filter through?